src/cdk/bidi/dir.ts

```diff
@@ -59,6 +64,8 @@ export class Dir implements Directionality, AfterContentInit, OnDestroy {
     return this.dir;
   }
 
+  readonly valueSignal = signal<Direction>('ltr');
```

Comment:
nit: it seems like this signal stores some info about a direction so I'm surprised that it named `valueSignal`

Reply:
This directive takes over the `[dir]` attribute, so the signal represents the value of the attribute. I named it to align with the pre-existing non-signal `value` property that was already here